wordpress-mobile/WordPress-Android
# Issue/6849 add signup methods

### Fix
Add panel that slides up when the ***Sign Up*** button is tapped on the prologue screen which displays the signup methods as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6849.  See the screenshots below for illustration.  The first screenshots show the button text change on the prologue screen.  The second screenshots show the signup panel in portrait orientation.  The third screenshots show the signup panel in landscape orientation.  Notice the third screenshots have a blank image for the ***Design*** screenshot since there was no design for the landscape orientation.

![auth_signup_screenshot_01_method_prologue](https://user-images.githubusercontent.com/3827611/33304074-e7bedd6e-d3c3-11e7-872c-83a786ad2523.png)

![auth_signup_screenshot_01_method_port](https://user-images.githubusercontent.com/3827611/33304078-ebdfbc56-d3c3-11e7-8be7-083a78c3731d.png)

![auth_signup_screenshot_01_method_land](https://user-images.githubusercontent.com/3827611/33304079-ee15b264-d3c3-11e7-9320-916aa38c2acd.png)

The landscape layout was derived from the prologue landscape layout.  The style used for the ***Terms of Service*** text is the same as the label text in the login flow.  The style used for the ***Sign Up with Email*** and ***Sign Up with Google*** buttons is the same as the ***Sign Up*** button on the prologue screen.

### Test
0. Clear app data.
1. Notice ***Sign Up*** button text.
2. Tap ***Sign Up*** button.
3. Notice signup options.
4. Tap ***Sign Up with Email*** button.
5. Notice signup screen is shown.

#### Note
Please, review the copy of the following text from the screenshots above.
> By choosing "Sign up," you agree to our <u>Terms of Service</u>.
